glide-core/telemetry/src/open_telemetry.rs

```diff
@@ -359,18 +363,30 @@ impl GlideSpan {
 pub struct GlideOpenTelemetryConfig {
     /// Default delay interval between two consecutive exports.
     flush_interval_ms: std::time::Duration,
-    /// Determines the protocol between the collector and GLIDE for traces
-    trace_exporter: GlideOpenTelemetrySignalsExporter,
-    /// Determines the protocol between the collector and GLIDE for metrics
-    metrics_exporter: GlideOpenTelemetrySignalsExporter,
+    traces: GlideOpenTelemetryTracesConfig,
+    metrics: GlideOpenTelemetryMetricsConfig,
+}
+
+#[derive(Clone, Debug)]
+pub struct GlideOpenTelemetryTracesConfig {
+    /// Determines the protocol between the collector and GLIDE
+    trace_exporter: Option<GlideOpenTelemetrySignalsExporter>,
+    /// The percentage of requests to sample and create a span for, used to measure command duration.
+    trace_requests_precentage: u32,
+}
+
+#[derive(Clone, Debug)]
+pub struct GlideOpenTelemetryMetricsConfig {
+    /// Determines the protocol between the collector and GLIDE
+    metrics_exporter: Option<GlideOpenTelemetrySignalsExporter>,
 }
 
 #[derive(Clone, Debug)]
 #[allow(dead_code)]
 pub struct GlideOpenTelemetryConfigBuilder {
     flush_interval_ms: std::time::Duration,
-    trace_exporter: GlideOpenTelemetrySignalsExporter,
-    metrics_exporter: GlideOpenTelemetrySignalsExporter,
+    traces_config: GlideOpenTelemetryTracesConfig,
+    metrics_config: GlideOpenTelemetryMetricsConfig,
```

Comment:
here it should be optional, not inside the struct itself. e.g.:
```suggestion
    traces_config: Option<GlideOpenTelemetryTracesConfig>,
    metrics_config: Option<GlideOpenTelemetryMetricsConfig>,
```

Also please add description to all of this struct and describe what happens if none of these traces_config, metrics_config are set